mock-core-configs/etc/mock/templates/centos-stream-9.tpl

```diff
@@ -1,4 +1,4 @@
-config_opts['chroot_setup_cmd'] = 'install tar gcc-c++ redhat-rpm-config redhat-release which xz sed make bzip2 gzip gcc coreutils unzip shadow-utils diffutils cpio bash gawk rpm-build info patch util-linux findutils grep'
+config_opts['chroot_setup_cmd'] = 'install tar redhat-rpm-config redhat-release which xz zstd sed make bzip2 gzip coreutils unzip shadow-utils diffutils cpio bash gawk rpm-build info patch util-linux findutils grep'
```

Comment:
Why zstd? I do not see it in 9's group `build`.

Reply:
zstd is the dependency of rpm-build and will be transitively loaded.
And I wonder why we do not have glibc-minimal-langpack here?

Reply:
We should, that's my mistake... Fixed!